src/inbox/tx.rs

```diff
@@ -139,24 +26,24 @@ impl<A, Rc: TxRefCounter> SetPriority for SendFuture<A, Rc> {
 }
 
 #[must_use = "Futures do nothing unless polled"]
-pub enum SendFuture<A, Rc: TxRefCounter> {
+pub enum SendFuture<A> {
     New {
         msg: SentMessage<A>,
-        tx: Sender<A, Rc>,
+        chan: Arc<Chan<A>>,
```

Comment:
I guess we should just replace this with `tx` again?

Reply:
You mean the variable name?

Reply:
I mean instead of holding the inner channel, we can just clone the sender, which is just chan + rc, right?

Reply:
There is no more `Sender` with this PR :)

Reply:
Ah, right! It would be `Address` then, I guess!

Reply:
I guess we could embed the `Address` yeah. Although that would be a cyclic dependency. It is not a problem but conceptually weird.

I am doing some other work on `SendFuture` atm so I'll go think about this once that is done.